### ISSUE TITLE
Problem running django with the newest version

### ISSUE THREAD

**Issue by SalahAdDin** [takeflight/wagtail-metadata#20]
Hi man

I haven't problem with old version, all worked fine; today i update it to newest version and i have this problem:
```bash
./manage.py makemigrations                                                                                                      1 ↵  4245  23:24:25 
/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/wagtail_embed_videos/models.py:21: RemovedInWagtail18Warning: The wagtail.wagtailadmin.taggable module is deprecated.
  from wagtail.wagtailadmin.taggable import TagSearchable
Traceback (most recent call last):
  File "./manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/core/management/__init__.py", line 341, in execute
    django.setup()
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/apps/config.py", line 199, in import_models
    self.models_module = import_module(models_module_name)
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/wagtailmetadata/models.py", line 13, in <module>
    IMAGE_MODEL = get_image_model()
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/wagtail/wagtailimages/models.py", line 373, in get_image_model
    image_model = apps.get_model(app_label, model_name)
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/apps/registry.py", line 192, in get_model
    self.check_models_ready()
  File "/home/salahaddin/Proyectos/uzman/lib/python3.5/site-packages/django/apps/registry.py", line 131, in check_models_ready
    raise AppRegistryNotReady("Models aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.
```

Actually i'm working with Wagtail `1.7rc1` and Django `1.10.2`.

Thanks.

**Comment by timheap** [comment]
torchbox/wagtail#3075 should help with this, although it will not be available until 1.8 is released sadly.

**Comment by SalahAdDin** [comment]
And so, what's current solution? :D
